services/src/main/java/io/druid/cli/CliHadoopIndexer.java

```diff
@@ -76,14 +75,10 @@ public void run()
         allCoordinates.add(DEFAULT_HADOOP_COORDINATES);
       }
 
-      final DefaultTeslaAether aetherClient = Initialization.getAetherClient(extensionsConfig);
-
       final List<URL> extensionURLs = Lists.newArrayList();
-      for (String coordinate : extensionsConfig.getCoordinates()) {
-        final ClassLoader coordinateLoader = Initialization.getClassLoaderForCoordinates(
-            aetherClient, coordinate, extensionsConfig.getDefaultVersion()
-        );
-        extensionURLs.addAll(Arrays.asList(((URLClassLoader) coordinateLoader).getURLs()));
+      for (final File extension : Initialization.getExtensionFilesToLoad(extensionsConfig)) {
```

Comment:
If extensions directory is empty, it will return a zero-length array of ```File[]```. Or did I miss something?